examples/00-mapdl-examples/spotweld.py

```diff
@@ -0,0 +1,75 @@
+"""
+.. _ref_spotweld:
+
+=================================================
+Running an input file - spotweld SHELL181 example
+=================================================
+
+This MAPDL example demonstrates how to model spot welding on three
+thin sheets of metal. Here, the full input file is simply run using
+the PyMAPDL interface.
+
+Using the following commands, you can directly use an APDL
+script within a PyMAPDL session with the following
+commands:
+
+"""
+
+from ansys.mapdl.core import launch_mapdl
+
+mapdl = launch_mapdl()
+if mapdl._local:
+    mapdl.upload("./spotweld/spot_weld.inp")
+    mapdl.input("spot_weld.inp")
+else:
+    mapdl.input("./spotweld/spot_weld.inp")
```

Comment:
I wonder if we should just put that `inp` file in the [example data repository](https://github.com/ansys/example-data)... This is fine though.

Reply:
Implemented in #[b1192fe](https://github.com/ansys/pymapdl/pull/2237/commits/b1192fefe16cd5f35fe10ca6585ad83ce07ead1c).